queue_services/entity-emailer/src/entity_emailer/worker.py

```diff
@@ -142,6 +143,20 @@ def process_email(email_msg: dict, flask_app: Flask):  # pylint: disable=too-man
         elif etype and etype == 'bc.registry.bnmove':
             email = bn_notification.process_bn_move(email_msg, token)
             send_email(email, token)
+        elif etype and etype == 'bc.registry.dissolution':
+            email = involuntary_dissolution_stage_1_notification.process(email_msg, token)
+            # Confirm the data.furnishingName
+            furnishing_name = email_msg['data']['furnishing']['furnishingName']
+            if furnishing_name not in involuntary_dissolution_stage_1_notification.PROCESSABLE_FURNISHING_NAMES:
+                raise QueueException('Furnishing name is not valid.')
+            try:
+                send_email(email, token)
+                # Update corresponding furnishings entry as PROCESSED
+                involuntary_dissolution_stage_1_notification.post_process(email_msg, 'PROCESSED')
```

Comment:
can we use the enums defined in the model?  https://github.com/bcgov/lear/blob/c4173d080519853112c66e7da59460ba9ff29d18/legal-api/src/legal_api/models/furnishing.py#L50-L55

Reply:
Yes, I can.
But, I didn't want to import the furnishing model at Work.py, because it was imported in the process.
So, I did like this at the post_process

`Furnishing.FurnishingStatus[status]`
Please check the code here. https://github.com/bcgov/lear/blob/ce97d714b16b5fe9a1577a0c99ad8d6538f5143f/queue_services/entity-emailer/src/entity_emailer/email_processors/involuntary_dissolution_stage_1_notification.py#L95

Anyway, if it doesn't look good to you, I will update it. :☺️

Reply:
yes, please update to import and use FurnishingStatus here and please update `involuntary_dissolution_stage_1_notification.post_process` as required.

Reply:
updated